client/trino-jdbc/src/main/java/io/trino/jdbc/AbstractTrinoResultSet.java

```diff
@@ -658,8 +658,8 @@ public Object getObject(int columnIndex)
         return column(columnIndex);
     }
 
-    @javax.annotation.Nullable
-    private static Object convertFromClientRepresentation(ClientTypeSignature columnType, @javax.annotation.Nullable Object value)
+    @jakarta.annotation.Nullable
```

Comment:
can this be imported?

Reply:
No. We have `Nullable` enum defined in the scope of this class already.

Reply:
```
public enum Nullable
{
    NO_NULLS, NULLABLE, UNKNOWN
}
```